players/src/dashboard/Dashboard.spec.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {render, fireEvent} from 'react-testing-library';
+import Dashboard from '../Dashboard/Dashboard';
+import 'jest-dom/extend-expect'
+
+describe("<Dashboard />", () => {
+    describe('Strike button', () => {
+        it("Fires strike event", () => {
```

Comment:
I like that you broke up your fire Events into their own tests, makes more sense to do each one individually. I've seen most people just making one big test w/ like 4 fire events in it.